tokens/testnet.json

```diff
@@ -41,5 +41,6 @@
       "description": "Testing USDC on Alephium",
       "logoURI": "https://raw.githubusercontent.com/alephium/tokens-meta/master/logos/TUSDC.png"
     }
-  ]
+  ],
+  "nftCollections": []
```

Comment:
we already have a collection on testnet right? we could add them directly.
I think it's this address https://testnet.alephium.org/addresses/28AL78tSb7apkhrmHSoxwt26i7jRD3uBDkbTqHtEmhnpf

Reply:
Yeah, it is not valid any more with the code change :)